sdk/communication/communication-common/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 2.1.2 (Unreleased)
```

Comment:
please use the unreleased 2.1.1 version (below)

Reply:
Done